docs/content/docs/user-docs/ack-tags.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Managing ACK Default Tags"
+description: "Managing ACK Default Tags"
+lead: ""
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 67
+toc: true
+---
+
+ACK controllers add default tags to AWS resources that support tagging.
+These default tags help in identifying and categorizing the AWS resources
+managed by ACK controllers.
+
+The two default tags added by ACK controller are `services.k8s.aws/controller-version`
+and `services.k8s.aws/namespace`. *controller-version* tag value is the name of
+corresponding AWS service and version for that controller(Ex: s3-v0.1.3).
+And the *namespace* tag value is the namespace for the ACK resource.(Ex: default)
+
+When tags are already present inside the resource spec, ACK default tags are
+added in AWS resource alongside the tags from resource spec. Priority is given to
+resource spec tags when there is a conflict between ACK default tag keys and tag
+keys in resource spec.
+
+Example:
+
+For a resource manifest like
+
+```yaml
+apiVersion: s3.services.k8s.aws/v1alpha1
+kind: Bucket
+metadata:
+  name: my-ack-tagging-bucket
```

Comment:
Please add `namespace: default` so people can directly see that the tag value matches the metadata value